packages/contentful--create-contentful-app/package.json

```diff
@@ -10,7 +10,7 @@
     "directory": "packages/contentful--create-contentful-app"
   },
   "engines": {
-    "node": ">=12",
+    "node": ">=14.14.0",
```

Comment:
Why starting at 14.14 and not 14.15?

Reply:
I thought I'd successfully run the command with node 14.14, but I just double checked and that isn't true.

Reply:
Right, which you were saying is a good proxy for what versions of node are being used, and thus what we should support. I remember now, thank you!